.github/workflows/cross-build.yaml

```diff
@@ -0,0 +1,32 @@
+name: Cloud Hypervisor Cross Build
```

Comment:
For 888028ab35350ba42d7984751789d6914995b15b please prefix first commit message line with "build: " and drop gerrit Change-Id